aries-site/src/pages/components/toast-notifications.mdx

```diff
@@ -0,0 +1,100 @@
+import { Example } from '../../layouts';
+import { ToastNotificationExample, ToastDiagram } from '../../examples';
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { StatusWarningSmall } from 'grommet-icons/icons/StatusWarningSmall';
+import { StatusUnknownSmall } from 'grommet-icons/icons/StatusUnknownSmall';
+import { Box } from 'grommet';
+
+<Example
+  docs="https://v2.grommet.io"
+  code={[
+      "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/visualizations/ToastNotificationExample.js",
+      "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/visualizations/ToastNotification.js",
+  ]}
+  figma="https://www.figma.com"
+>
+  <ToastNotificationExample />
+</Example>
+
+## Guidance
+
+Everything that you need to know about toasts and more.
+
+### When should you use a Toast Notification?
+
+Toasts should be considered when displaying these types of information to the user:
+
+- Low attention messages that do not require user action
+- Singlular status updates
+- Confirmations
+- Information that does not need to be followed up
+
+Do not use Toasts if the information contains the following:
+
+- High attention and crtitical information
+- Time-sensitive information
+- Requires user action or input
+- Batch updates
+
+Due to their limited use cases, toasts are advised to be used sparingly or even substituted entirely for a different type of notification. In no way are toasts mandatory elements for a platform’s notification system.
+
+## Anatomy 
+
+<Box pad="small">
+  <ToastDiagram />
+</Box>
```

Comment:
Can we eliminate Box, add pad to ToastDiagram and ...rest in ToastDiagram implementation?